packages/lit-dev-content/site/docs/ssr/server-api.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Lit SSR server API
+eleventyNavigation:
+  key: 🧪 Server API
+  parent: Server-side rendering
+  order: 2
+---
+
+<details class="pre-release">
+  <summary> 🧪 Pre-release software: not for production.</summary>
+  
+  Lit labs packages are published to get feedback from the wider community. This code shouldn't be used in production, but we encourage you to try it out and [file issues](https://github.com/lit/lit/issues/new/choose) for any bugs you find. For general feedback, please use the GitHub [discussion](https://github.com/lit/lit/discussions).
```

Comment:
We should make a shortcode/ helper so this isn't repeated and can be updated centrally.

Reply:
Was gonna say the same thing; we have a tracking issue in https://github.com/lit/lit.dev/issues/838) I also think we should use a more more general statement with a link to a more detailed description on the Labs page (rather than repeating the caveats on each page that references them), something like:

> This package is part of the Lit Labs family of experimental packages. See the [Lit Labs](#) page for guidance on using Labs software in production."

And I think we need to workshop the contents of the detailed description. Specifically, I'm not sure about saying "This code _shouldn't_ be used in production", because I think we do want to encourage not just tire kicking but actual real-world usage; the caveat is that it needs to be done with full understanding that breaking changes are more likely, and comes with the possibility of being end-of-lifed if it doesn't end up being generally useful enough.